examples/containerd-cluster/hosts.yml

```diff
@@ -1,5 +1,15 @@
+[bastion]
```

Comment:
Or, if we should have a separate file which has the template for clusters that have a bastion set-up...